README.md

```diff
@@ -70,5 +72,5 @@ To generate HTML, run `GTFSRealtimeViz.visualize/1`. This will return a `String.
 If you just want a standalone file to open in a browser, you might run:
 
 ```ex
-iex> File.write!("/path/to/file.html", GTFSRealtimeViz.visualize(:prod))
+iex> File.write!("/path/to/file.html", GTFSRealtimeViz.visualize(:prod, %{"Route" => {"First stop", "123", "124"},  {"Second stop" , "125", "126"}]))
```

Comment:
Looks like you're missing the opening square bracket here.

Reply:
you know, when youre right youre right.  and you? youre always right.

i actually know exactly what happened too, i had my hand in the wrong spot on my keyboard and vim did its thing.  maybe the sublime people were right all along

Reply:
lol